dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumRawEvent.kt

```diff
@@ -169,4 +169,8 @@ internal sealed class RumRawEvent {
         val target: String,
         override val eventTime: Time = Time()
     ) : RumRawEvent()
+
+    internal data class SendNow(
```

Comment:
Let's name it `SendActionNow` or even `SendCustomActionNow`. `SendNow` is ambiguous as it could mean send the view now, or send the resource now or …

Reply:
makes sense, will do